ckine/figures/figure3.py

```diff
@@ -63,14 +64,14 @@ def makeFigure():
 
     plot_cells(ax[7], tl.to_numpy(factors_activ[1]), 1, 2, cell_names)
     plot_cells(ax[8], tl.to_numpy(factors_activ[1]), 1, 3, cell_names)
-
     legend = ax[7].get_legend()
     labels = (x.get_text() for x in legend.get_texts())
     ax[4].legend(legend.legendHandles, labels, loc='center', prop={"size": 9})
-
     ax[7].get_legend().remove()
     ax[8].get_legend().remove()
-
+    circle = Line2D([], [], color='black', marker='o', linestyle='None', markersize=6, label='Variance of Model')
```

Comment:
Why not just put this label on the y-axis? There's no need to refer to this as `R2X`.

Reply:
Alright, took care of this one. Should be ready to go if you're happy with the rest!